src/setup.py

```diff
@@ -1,15 +1,15 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
 setup(
     name="snowalert-runners",
     version='1.0',
-    packages=['runners', 'runners.helpers', 'runners.plugins'],
+    packages=find_packages(),
```

Comment:
these should go away if you rebase

Reply:
Ran git fetch and git rebase.

Reply:
`git fetch` updates your local db version of remote and `git rebase` rebases on the remote. he, I meant `git rebase v1.5.5`, which rebases this pull against the latest v1.5.5